integreat_cms/cms/views/push_notifications/push_notification_form_view.py

```diff
@@ -379,3 +393,27 @@ def extract_pn_details(request, push_notification):
         "all_languages": all_languages,
         "disable_edit": disable_edit,
     }
+
+
+def patch_missing_regions_field(request):
```

Comment:
I think this is the wrong way of handling this problem. I'd prefer it if the value was contained in the html form already, and only hidden from the user via css.

Reply:
Yes, but I figured we needed to validate it in this case anyway (whether the user has access to it). Do you have a quick suggestion for an alternative approach?

Reply:
In my opinion your suggestion looks valid @PeterNerlich , thanks. I would rephrase the logged message a bit (could be a bug, not a manipulation):
```python
            logger.warning(
                "%r is not allowed to post news to %r (allowed regions: %r)",
                request.user,
                push_notification_instance,
                request.available_regions
            )
            raise PermissionDenied
```